devops/.env.sample

```diff
@@ -7,10 +7,11 @@ ACR_NAME=__CHANGE_ME__
 
 # Azure Resource Manager credentials used for CI/CD pipelines
 ARM_SUBSCRIPTION_ID=__CHANGE_ME__
+
 # If you want to deploy using your own (logged-in) credentials, remove the next 3 vars
```

Comment:
Now that you reversed this, I think the comment should say "If you want to deploy using GitHub workflow, uncomment and fill the values for the following 3 variables". Also, `__CHANGE_ME__` could've remained there.

Reply:
done